Dalamud.DiscordBridge/Plugin.cs

```diff
@@ -27,8 +29,26 @@ public void Initialize(DalamudPluginInterface pluginInterface)
             this.Config = (Configuration)this.Interface.GetPluginConfig() ?? new Configuration();
             this.Config.Initialize(this.Interface);
 
+            
+
             this.Discord = new DiscordHandler(this);
-            this.Discord.Start();
+            //Task t = this.Discord.Start();
+            Task.Run(async () =>
+            {
+                await this.Discord.Start();
+            });
+
+            // try to force-reload it
+            if (!this.Discord.IsConnected)
```

Comment:
Does this actually do anything? Try to breakpoint - you'll likely see that this gets ran every time, since the Task.Run from before didn't even get to run yet at that point.

The first one may be enough.

Reply:
It can be cut down a bit. Some of that is leftovers from debugging.

With just `Task t = this.Discord.Start();`, the bot never actually starts when Dalamud loads the plugin normally. (See multiple users reporting this in issues on discord). Using Task.Run() solved it, although the check to try again can be safely removed.